mail/components/default-layout.js

```diff
@@ -0,0 +1,61 @@
+import { Body } from '@react-email/body';
+import { Container } from '@react-email/container';
+import { Head } from '@react-email/head';
+import { Heading } from '@react-email/heading';
+import { Html } from '@react-email/html';
+import { Img } from '@react-email/img';
+import { Preview } from '@react-email/preview';
+import { Text } from '@react-email/text';
+
+import webserver from '../../infra/webserver';
```

Comment:
Esse `import webserver from '../../infra/webserver'` contradiz a ideia de isolar as dependências.

Reply:
Sim, adicionei esse trecho apenas para conseguir a url da logo, vou pensar uma solução melhor.

Reply:
> Sim, adicionei esse trecho apenas para conseguir a url da logo, vou pensar uma solução melhor.

Se deixar o módulo responsável apenas pelo design, isso será passado junto com os parâmetros. Se for um módulo separado, acredito que seja a melhor forma.